src/atoms/Grid.js

```diff
@@ -1 +1,27 @@
-export { Grid } from 'react-styled-flexboxgrid';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import config, { DIMENSION_NAMES } from '../utils/grid-config';
+
+const Grid = styled.div`
+  margin-right: auto;
+  margin-left: auto;
+  padding-right: ${p => `${config(p).outerMargin}rem`};
+  padding-left: ${p => `${config(p).outerMargin}rem`};
+  ${p => !p.fluid && css`
+  ${DIMENSION_NAMES.map((t) => {
+		return config(p).container[t] && config(p).media[t]`
+       width: ${config(p).container[t]}rem;
```

Comment:
`Col`, `Row`, `Grid` and `grid-config`, is all taken from `react-styled-flexboxgrid` - almost no changes, could be merged upstream!  

I think the related changes is inside this map function that start on line 12